controllers/badge.js

```diff
@@ -1,4 +1,5 @@
 const badgeQuery = require('../models/badges')
+const logger = require('../utils/logger')
```

Comment:
We don't need to import logger. It is added to global object I suppose

Reply:
Got it!